File: QID-1529-MVAscabank45/MVAscabank45.r

# clear all variables
rm(list = ls(all = TRUE))
graphics.off()

# load data
x = read.table("bank2.dat")

# plot
plot(x[, 4], x[, 5], ylab = "", xlab = "", ylim = c(7, 13), xlim = c(7, 13))
title("Swiss bank notes")

File: QID-1529-MVAscabank45/MVAscabank45.py
import pandas as pd
import matplotlib.pyplot as plt

x = pd.read_csv("bank2.dat", sep = "\s+", header=None)

fig, ax = plt.subplots(figsize = (10, 10))
ax.scatter(x.iloc[:,3], x.iloc[:,4], c = "w", edgecolors = "black")
plt.xlim(7, 13)
plt.ylim(7, 13)
plt.title("Swiss bank notes")
plt.show()